### PR TITLE
[DC-2364] Generate the wear_study table

### PR DESCRIPTION
**Pull request by brendagutman**

### CODE REVIEW

File: data_steward/cdr_cleaner/cleaning_rules/generate_wear_study_table.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+JIRA_ISSUES = ['DC2364']
+
+CREATION_QUERY = JINJA_ENV.from_string("""
+CREATE OR REPLACE TABLE `{{project_id}}.{{dataset_id}}.wear_study` AS
+SELECT
+  DISTINCT person_id,
+  'Yes' AS resultsconsent_wear,
+  MIN(CASE WHEN value_source_concept_id = 2100000009 THEN observation_date END) AS wear_consent_start_date,
+  MIN(CASE WHEN value_source_concept_id = 2100000008 THEN observation_date END) AS wear_consent_end_date
```
Comment (nishanthpp93) [inline data_steward/cdr_cleaner/cleaning_rules/generate_wear_study_table.py:26]: Interesting, this uses the earliest response in both cases.

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generate_wear_study_table_test.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        cls.fq_sandbox_table_names = []
+
+        # call super to set up the client, create datasets
+        cls.up_class = super().setUpClass()
+
+    def setUp(self):
+        # create tables
+        super().setUp()
+
+        self.date_one = parser.parse('2021-01-01').date()
```
Comment (nishanthpp93) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generate_wear_study_table_test.py:59]: I like this! Good idea

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generate_wear_study_table_test.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                  person_id,
+                  observation_concept_id,
+                  observation_source_concept_id,
+                  value_source_concept_id,
+                  observation_date,
+                  observation_type_concept_id
+                  )
+                VALUES
+                  -- one yes record --
+                  (1,1,1,2100000010,2100000009,date('2021-01-01'),1),
```
Comment (nishanthpp93) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generate_wear_study_table_test.py:75]:
```suggestion
                  (1,1,1,2100000010,2100000009,date('{{self.date_one}}'),1),
```

**Review by nishanthpp93 — APPROVED** [review]
Overall looks good! Added a minor suggestion but not required for merge.

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generate_wear_study_table_test.py (hunk 3 above)
Comment (brendagutman) [reply tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generate_wear_study_table_test.py:75]: Going to merge so we can hopefully have it in the data refresh.

**Pull request merged**